dp/cloud/python/magma/radio_controller/services/dp/service.py

```diff
@@ -61,11 +72,15 @@ def GetCBSDState(self, request: CBSDRequest, context) -> CBSDStateResult:
             CBSDStateResult: State result with RF data
         """
         logger.info(f"Getting CBSD state for {request.serial_number=}")
+        message_type = 'GetCBSDState'
         with self.session_manager.session_scope() as session:
             cbsd = session.query(DBCbsd).filter(
                 DBCbsd.cbsd_serial_number == request.serial_number,
             ).first()
-            self._log_request(session, 'GetCBSDState', request, cbsd)
+            try:
+                self._log_request(message_type, request, cbsd)
+            except (requests.HTTPError, requests.RequestException, TypeError) as err:
```

Comment:
IMO redo exception handling.
Theoretically fluentd integration doesn't have to use http ingestion. IMO it would be cleaner if the fluentd_client had its own exception class, and encapsulated the usage or `requests` there.

Reply:
d

Reply:
done